### PR TITLE
[GITHUB-1] Creates Prometheus Server role

### PR DESCRIPTION
**Pull request by quater**

### CODE REVIEW

File: templates/prometheus_server.default.j2 [hunk 1]
```diff
@@ -0,0 +1,12 @@
+CONFIG={{ prometheus_server.path.config }}/prometheus.yml
```
Comment (racer1680) [inline templates/prometheus_server.default.j2:1]: I can't find path.config. What is it?

File: README.md [hunk 2]
```diff
@@ -1 +1,106 @@
-# prometheus_server
+# Prometheus Server
```
Comment (lobsterdore) [inline README.md]: This should be prometheus_server to match the role name.

File: templates/prometheus_server.default.j2 (hunk 1 above)
Comment (lobsterdore) [reply templates/prometheus_server.default.j2:1]: It's in defaults, line 13.

File: defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  opts: ~
+  path:
+    config: /home/prometheus/config
+    data: /var/lib/prometheus/db
+    install: /home/prometheus
+    log: /var/log/prometheus
+    pid: /var/run/prometheus
+    rule: /home/prometheus/rules
+    sd: /home/prometheus/tgroups
+  start_on_boot: yes
```
Comment (lobsterdore) [inline defaults/main.yml]: I don't think we need this setting for the server, just the exporters.

File: defaults/main.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    url: ~
+  checksum: sha256:4779d5cf08c50ed368a57b102ab3895e5e830d6b355ca4bfecf718a034a164e0
+  download_dest: /tmp
+  download_url: https://github.com/prometheus/prometheus/releases/download/v1.7.1/prometheus-1.7.1.linux-amd64.tar.gz
+  environment: dev
+  group: prometheus
+  opts: ~
+  path:
+    config: /home/prometheus/config
+    data: /var/lib/prometheus/db
```
Comment (lobsterdore) [inline defaults/main.yml]: Can you move data and logs to the /home/prometheus directory? I am not so fussed about the run directory but generally we try to pull all directories into the dedicated service user if possible.

**Review by lobsterdore — APPROVED** [review]
LGTM 👍

**Comment by lobsterdore** [comment]
With a previso that the config files will need to redone when we integrate this into -dev.

**Pull request merged**